taxonomy-editor-frontend/src/pages/editentry/ListAllNonEntryInfo.jsx

```diff
@@ -0,0 +1,103 @@
+import { Box, Paper, Stack, TextField, Typography } from "@mui/material";
+import { getIdType } from "./createURL";
+import ISO6391 from 'iso-639-1';
+
+/** 
+ * Parent component used for rendering info on a stopword, synonym, header or footer
+ * If node is an "stopword/synonym": Stopwords/synonyms, language and comments are rendered
+ * If node is "header/footer": Only comments are rendered
+*/
+
+const ListAllNonEntryInfo = ({ nodeObject, id, setNodeObject }) => {
+    
+    // TODO: Change variables to state variables and wrap Object.keys() inside a useEffect()
+
+    // Stores 2 letter language code (LC) of the tags
+    let languageCode = '';
+    // Storing keys and values that needs to be rendered for editing
+    let renderedNonEntryInfo = {}
+
+    if (nodeObject) {
+        Object.keys(nodeObject).forEach((key) => {
+            /** 
+             * Get all tags and its corresponding language code
+             * Tagids need to be recomputed, so shouldn't be rendered
+             * Eg: tags_fr
+            */ 
```

Comment:
nit: this comment can be normal comment `//`, the benefit of having the doc string is for functions (components, etc) is that some IDEs show you the doc when using it